app/models/publication.rb

```diff
@@ -536,6 +542,10 @@ def doi_url_path
         field(:preferred_file_version_none_email_sent)
       end
     end
+
+    scope do
```

Comment:
Looks good. Did the omission of this scope cause a test to fail?

Reply:
Omission actually caused the export process to fail. The tests failed because I originally had `joins` instead of `left_joins`